### PR TITLE
rbenv 1.3.0

### PR DESCRIPTION
**Pull request by mislav**
https://github.com/rbenv/rbenv/releases/tag/v1.3.0

### CODE REVIEW

File: Formula/r/rbenv.rb [hunk 1]
```diff
@@ -1,11 +1,16 @@
...
+  url "https://github.com/rbenv/rbenv/archive/refs/tags/v1.3.0.tar.gz"
+  sha256 "7e49e529ce0c876748fa75a61efdd62efa2634906075431a1818b565825eb758"
   license "MIT"
   head "https://github.com/rbenv/rbenv.git", branch: "master"
 
+  livecheck do
+    url :stable
+    strategy :github_latest
+  end
+
```
Comment (p-linnane) [inline Formula/r/rbenv.rb:13]: Thanks for the PR @mislav! Are you planning on using GitHub pre-releases? If not, we can omit this and livecheck will use the git tags automatically.

**Review by branchv — APPROVED** [review]

**Comment by jasonkarns** [comment]
@mislav do you have any recollection why the bash completions aren't installed here using `bash_completion` helper?

I would have expected `bash_completion.install "completions/rbenv.bash"`. Without it, `brew install rbenv` doesn't report that completions were installed.

nodenv's formula _does_ use the bash_completion helper. And correspondingly, _does_ get a caveat to that effect when installing nodenv:

> ==> Caveats
> Bash completion has been installed to:
>  /opt/homebrew/etc/bash_completion.d

I posed this question on the PR when updating nodenv, to see if rbenv's behavior was intentional; and if so, why. That way nodenv can behave similarly if the same reasons apply. https://github.com/Homebrew/homebrew-core/pull/231397#discussion_r2236530864

Do you have any insight into this? (apologies for the direct [commit comment](https://github.com/Homebrew/homebrew-core/commit/5d36ddba643bc5742615dd0bf343e97201f98361#commitcomment-163017554) earlier)

cc/ @SMillerDev

**Comment by mislav** [comment]
I've answered first here before I saw the PR notification https://github.com/Homebrew/homebrew-core/commit/5d36ddba643bc5742615dd0bf343e97201f98361#commitcomment-163026158

**Comment by jasonkarns** [comment]
thank you!

**Pull request merged**